public/components/notebooks/components/__tests__/notebook.test.tsx

```diff
@@ -73,7 +82,14 @@ describe('<Notebook /> spec', () => {
     const cloneNotebook = jest.fn();
     const deleteNotebook = jest.fn();
     const setToast = jest.fn();
-    const location = jest.fn();
+    jest.mock('react-router', () => ({
```

Comment:
This might be able to be removed as `useLocation` is not necessary b/c `location` is passed into SPA from OSD-CoreStart object